Resources/Locale/en-US/criminal-records/general-criminal-records.ftl

```diff
@@ -0,0 +1,46 @@
+general-criminal-record-console-window-title = Criminal Records Computer
+general-criminal-record-console-select-record-info = Select a record on the left.
+general-criminal-record-console-empty-state = No records found!
+general-criminal-record-console-no-record-found = No record was found for the selected person.
```

Comment:
Probably can remove the `general-` prefix from all of these. It would be good if you can shorten the prefix `criminal-record-console` somehow too.

Reply:
I just copied it from station records and replace `station` with `criminal`. I can do like `crim-records` if it's important to shorten it